app/views/moderations/actions_panel.html.erb

```diff
@@ -50,7 +50,9 @@
       <span></span>
       <%= inline_svg_tag("checkmark.svg", aria: true, class: "crayons-icon vomit-checkmark", title: "Checkmark") %>
     </button>
-    <span class="how-this-works-section">How does this work?</span>
+    <a href="<%= URL.url %>/community-moderation" target="_blank">
```

Comment:
`target="_blank"` links should also have `rel="noopener"`: https://dev.to/dhilipkmr/why-should-you-use-noopener-beware-of-security-flaws-3i57